src/containers/Tenant/Diagnostics/Partitions/Partitions.scss

```diff
@@ -5,11 +5,12 @@
     flex-grow: 1;
 
     height: 100%;
+    margin-right: -20px;
 
     @include mixins.flex-container();
 
     &__controls {
```

Comment:
It looks like you do not need this class anymore, you also do not need properties in `.ydb-diagnostics-partitions` class